### PR TITLE
Add the Asset Versioning section

### PR DESCRIPTION
**Pull request by jonbellah**
Covers a basic process for keeping assets properly versioned. Addresses #136

### CODE REVIEW

File: _includes/markdown/PHP.md [hunk 1]
```diff
@@ -521,6 +521,20 @@ if ( current_theme_supports( 'custom-js-feature' ) ) {
...
+It is recommended that engineers use a constant to define their theme or plugin version, then reference that constant when using registering scripts or styles. For example:
+
+```php
+<?php
+define( 'THEME_VERSION', '0.1.0' );
+
+wp_register_script( 'custom-script', get_template_directory_uri() . '/js/asset.js', array(), THEME_VERSION );
+```
+
+Remember to increment the version in the defined constant prior to deployment.
```
Comment (stevegrunwell) [inline _includes/markdown/PHP.md:536]: It's probably worth adding a disclaimer that assets that already have real versions (e.g. version 0.5.0 of js/vendor/my-cool-library.js) should use the version number from the library, not the theme asset version. This avoids the cache from needlessly being flushed when underlying vendor script hasn't actually changed.

Comment (ericmann) [reply _includes/markdown/PHP.md:536]: 👍

Comment (jonbellah) [reply _includes/markdown/PHP.md:536]: Good thinkin'... I'll add that.

**Comment by tlovett1** [comment]
Nice!

**Pull request merged**